mdast_util_to_markdown/src/message.rs

```diff
@@ -0,0 +1,24 @@
+use core::{error::Error, fmt::Display};
+
+use alloc::string::{String, ToString};
+
+#[derive(Debug, PartialEq)]
+pub struct Message {
+    pub reason: String,
+}
+
+impl Error for Message {}
+
+impl Display for Message {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(f, "{}", self.reason)
+    }
+}
+
+impl From<&str> for Message {
+    fn from(value: &str) -> Self {
+        Message {
+            reason: value.to_string(),
+        }
+    }
+}
```

Comment:
I think there’s an error struct now in this project, since https://github.com/wooorm/markdown-rs/pull/114. Which models the JS version `vfile-message`. Could that be used here too?

Reply:
Yes sure

Reply:
I updated it let me know if the changes are fine here